test/tstQueryTreeManufacturedSolution.cpp

```diff
@@ -255,11 +263,13 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(structured_grid, TreeTypeTraits,
   offset_host = Kokkos::create_mirror_view(offset);
   Kokkos::deep_copy(offset_host, offset);
 
+  ref_counts = std::vector<int>(n);
+  zeros = std::vector<int>(n);
   for (int i = 0; i < n; ++i)
-  {
-    BOOST_TEST(offset_host(i) == i);
-    BOOST_TEST(ref[i].count(indices_host(i)) != 0);
-  }
+    ref_counts[i] = ref[i].count(indices_host(i));
+
+  BOOST_TEST(offset_host == offset_ref, tt::per_element());
+  BOOST_TEST(ref_counts != zeros, tt::per_element());
```

Comment:
and here (except for `offset_host/offset_ref` since that doesn't require defining more variables).

Reply:
Not a big deal for me (and I didn't request changes).